docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+if [ "${1#-}" != "${1}" ] || [ -z "$(command -v "${1}")" ]; then
```

Comment:
That bash magic 😁

Reply:
There are 2 checks in this bash script.
the first check checks if there are `-` in the first argument like so:
```bash
1=foo
echo $1     # output: foo
echo ${1#-} # output: foo

1=--foo       
echo $1     # output: --foo
echo ${1#-} # output: -foo
```
the second check checks if the first argument is a command that exists in the system.
the flag `-z` check if there is an output. if the command is found on the system `command -v "${1}"` will return its path (!true) if not it will return nothing (!false).

TLDR:
`IF` (is the first character in the first argument is `-`) `OR` (`NOT`(is the first argument a command)) `THEN` run dockly

Reply:
Splendid, thanks!